### PR TITLE
Solution:

### PR DESCRIPTION
**Pull request by nikolasdymchenko7**
Solution:
- Created new classes: Ball and Lottery
- Created Enum: Colors
- Added implementation to the method: getRandomColor()
- Added method getRandomBall to the Lottery class
- Created three balls in the Application class and printed them

### CODE REVIEW

File: src/main/java/core/basesyntax/Colors.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public enum Colors {
```
Comment (okuzan) [inline src/main/java/core/basesyntax/Colors.java:3]: enum should be in singular

File: src/main/java/core/basesyntax/Lottery.java [hunk 2]
```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int BOUNDARY_RANDOM = 100;
+
+    public Ball getRandomBall() {
+        int generatedNumber = new Random().nextInt(BOUNDARY_RANDOM);
```
Comment (okuzan) [inline src/main/java/core/basesyntax/Lottery.java:9]: create Random and ColorSupplier at class level

**Review by okuzan — CHANGES_REQUESTED** [review]

File: src/main/java/core/basesyntax/Application.java [hunk 3]
```diff
@@ -1,7 +1,14 @@
 package core.basesyntax;
 
 public class Application {
+    private static final int NUMBER_OF_LOTTERIES = 3;
+
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Lottery[] lotteries = new Lottery[NUMBER_OF_LOTTERIES];
+        for (int i = 0; i < NUMBER_OF_LOTTERIES; i++) {
+            lotteries[i] = new Lottery();
```
Comment (okuzan) [inline src/main/java/core/basesyntax/Application.java:10]: don't use array at all
you need only one lottery

**Review by okuzan — CHANGES_REQUESTED** [review]

File: src/main/java/core/basesyntax/Lottery.java (hunk 2 above)
Comment (nikolasdymchenko7) [reply src/main/java/core/basesyntax/Lottery.java:9]: @okuzan Sorry, could you explain why you need to create instances of Random and ClassSupplier at the class level?

**Pull request closed without merging**